templates/iam_policy/disaster_recovery_policy.json.tpl

```diff
@@ -40,6 +40,11 @@
         "arn:aws:logs:eu-west-2:${account_id}:log-group:/disaster-recovery"
       ],
       "Sid": "writeLogs"
+    },
+    {
+      "Action" : "ecr:GetAuthorizationToken",
+      "Effect": "Allow",
+      "Resource": "*"
```

Comment:
Should we not limit the resources?

Reply:
`GetAuthorizationToken` works with "*". You can't restrict it to certain resources.